doc/switch-components.md

```diff
@@ -0,0 +1,56 @@
+# Controlled Switch Component
+https://reactjs.org/docs/forms.html#controlled-components
```

Comment:
@petrKavulok I think those links are not necessary here. Its kinda clear what is controlled and uncontrolled switch here in our case